context/app/static/js/components/Routes/Route.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Container from '@material-ui/core/Container';
+
+const StyledContainer = styled(Container)`
+  margin-top: ${(props) => (props.$mt ? props.theme.spacing(2) : 0)}px;
+`;
+
+function Route({ children, mt, constrainWidth }) {
+  const constrainWidthProps = constrainWidth ? { maxWidth: 'lg' } : { maxWidth: false, disableGutters: true };
+
+  return (
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    <StyledContainer $mt={mt} {...constrainWidthProps}>
+      {children}
+    </StyledContainer>
+  );
+}
+
+Route.propTypes = {
+  mt: PropTypes.bool,
+  constrainWidth: PropTypes.bool,
+};
+
+Route.defaultProps = {
+  mt: false,
+  constrainWidth: false,
+};
+
+export default Route;
```

Comment:
I'm not sure that `Route` is the clearest name for this... I think of a route as something like a (path, action) pair, and neither of those are here, though maybe the children are the action?

Reply:
- Naming the component `Route` is similar to patterns I've seen people use with `react-router`.
- I thought it would make sense to be more explicit about when something is being constrained vs. not, but I can make the switch if you think it's worthwhile.
- Error pages do not use `mt` and we will likely have additional pages which don't in the future.